### DISCUSSION TITLE
Non-Continous Tag writing

### DISCUSSION THREAD

**Discussion by Saldef** [standardnotes/forum#2927] [General]
When I Tag my note, then hit enter, the editing cursor for the next tag disappears, so I need to click again on the Tag menu under the note title each time for each tag. That is a lot of friction for me, I like to stay on my keyboard all the time, going back to the mouse each time I want to add a new tag is annoying.

I am on Windows 10, App version 3.132.2 (Latest stable I guess). 
Is this already known? any plan to fix it?

**Comment by effieeee** [comment]
Haven’t yet confirmed of this is intended on the app, but I’ve raised it internally just in case!
